app/assets/javascripts/govuk_publishing_components/analytics/analytics.js

```diff
@@ -49,7 +54,7 @@
   Analytics.prototype.trackPageview = function (path, title, options) {
     arguments[0] = arguments[0] || this.defaultPathForTrackPageview(window.location)
     if (arguments.length === 0) { arguments.length = 1 }
-    this.sendToTrackers('trackPageview', this.pii.stripPII(arguments))
+    return this.sendToTrackers('trackPageview', this.pii.stripPII(arguments))
```

Comment:
Why this change?

Reply:
@andysellick `sendToTrackers` isn't a public function, so I can't access it in the tests. Therefore I return the value of it in the functions that are publicly accessible `trackPageview / trackEvent / trackShare` so that I can verify in the tests that analytics is disabled.

Reply:
Just to check, this doesn't impact the outcome? As in, does returning `this.sendToTracker` instead of executing it still call that function?

Reply:
@andysellick It's still being executed, the result of the execution is what is returned. If it didn't have `()` on the function, then the function would be being returned instead of executed.

Example here if you want to play around with it: https://jsfiddle.net/spLfjez5/2/

Reply:
Cool, no worries, just wanted to be sure 👍